src/Data/Unit.elm

```diff
@@ -58,6 +58,13 @@ impactToFloat (Quantity value) =
     value
 
 
+impactPefScore : Impact -> Ratio -> Impact -> Impact
+impactPefScore normalization weighting =
+    Quantity.divideBy (impactToFloat normalization)
+        >> Quantity.multiplyBy (ratioToFloat weighting)
+        >> Quantity.multiplyBy 1000
```

Comment:
Maybe add a comment here explaining why this multiplication?

Reply:
Addressed by 7db578e9990af05cf5cda42a36cf06be041506a0